packages/sdk/src/db_connection_impl.ts

```diff
@@ -685,6 +730,7 @@ export class DbConnectionImpl<
             ?.emitter.emit('error', errorContext, error);
         } else {
           console.error('Received an error message without a queryId: ', error);
+          // TODO: This should actually kill the connection.
```

Comment:
Should it? Maybe just add to the comment with a bit about why.

Reply:
I added a comment explaining why we need to kill the connection.